src/Literate.jl

```diff
@@ -670,24 +668,20 @@ end
 
 # Execute a code-block in a module and capture stdout/stderr and the result
 function execute_block(sb::Module, block::String)
-    # r is the result
-    # status = (true|false)
-    # _: backtrace
-    # str combined stdout, stderr output
-    r, status, _, str = Documenter.withoutput() do
+    c = IOCapture.iocapture(throwerrors=false) do
```

Comment:
The `withoutput` version here that I am removing doesn't actually check for `InterruptException`. But, arguably, it would be better if it did I think?

```suggestion
    c = IOCapture.iocapture(throwerrors = :interrupt) do
```